src/Configuration/CFGCommon/Test/CFGCommon_test.cpp

```diff
@@ -68,7 +68,6 @@ void test_arg() {
   CFG_ASSERT(errors[3] ==
              "Can only specify maximum of 2 argument(s), but found 4 "
              "argument(s) is specified");
-  arg.print();
```

Comment:
Why remove this?

Reply:
This creates a lot of print unnecessary of print-out messages in unit tests. In unit testing, printing out the pass or fail message is sufficient. The `arg.print()` is useful during debugging.